core/feconf.py

```diff
@@ -1080,8 +1081,12 @@ def get_empty_ratings() -> Dict[str, int]:
 CLASSROOM_ID_TO_NAME_HANDLER_URL = '/classroom_id_to_name_handler'
 UNUSED_TOPICS_HANDLER_URL = '/unused_topics'
 NEW_CLASSROOM_ID_HANDLER_URL = '/new_classroom_id_handler'
+NEW_CLASSROOM_HANDLER_URL = '/classroom_admin/create_new'
+ALL_TOPICS_CLASSROOM_INFO_HANDLER_URL = (
+    '/all_topics_classroom_info_handler'
+)
+ALL_CLASSROOMS_SUMMARY_HANDLER_URL = '/all_classrooms_summary_handler'
```

Comment:
Same here, all and handler are implied, so I think /classrooms_summary should suffice

Reply:
Done.